pkg/webui/console/components/user-data-form/index.js

```diff
@@ -179,29 +223,55 @@ class UserForm extends React.Component {
           name="state"
           component={Select}
           options={approvalStateOptions}
+          required
         />
         <Form.Field
           title={sharedMessages.admin}
           name="admin"
           component={Checkbox}
           label={m.adminLabel}
         />
+        {!update && (
```

Comment:
Actually, it would be handy to be able to change passwords of existing users as well, no? I don't know whether the API allows that though, did you check that?

Reply:
looks like the API supports it, but you have to provide the old password, so it probably needs a slightly different form. i can do a update/forgot password as another PR